docs/en/8-MicroSim/GettingStarted.md

```diff
@@ -0,0 +1,125 @@
+
+# Getting started with Kubeflow Notebooks and OpenM++.
+
+This document is intended to act as an introduction to The AAWs Kubeflow Notebook servers and their use for the OpenM++ project.
+
+
+## Starting
+
+To access the AAW Kubeflow portal, navigate to the following website.
+
+[https://kubeflow.aaw.cloud.statcan.ca/](https://kubeflow.aaw.cloud.statcan.ca/) 
+
+This will redirect you to a Microsoft log-in page.
+
+![Login01](../images/Login01.png)
+
+Select the account you wish to use and proceed with the authentication.
+
+After your credentials are authenticated, you will be redirected to the AAW Kubeflow management panel. 
+
+![Kube Flow Management Panel](../images/KFMP01.png)
+
+## Create a notebook.
+
+Click on the Create a New Notebook server button.
+
+![Create Notebook01](../images/CreateNB01.png)
+
+This brings up the new Notebook screen.
+
+To create a new Notebook, three things must be set:
+- Ensure the correct Namespace is selected.
+- A unique name in the name field. A unique name is automatically generated in the Name field when you click in it,
```

Comment:
Is this right? I've never had a unique name get automatically generated for me, and trying it now I cannot seem to get it to do so (unless I'm doing something wrong)

Reply:
Noted.